CHANGELOG.md

```diff
@@ -99,10 +99,11 @@ For status on this, see https://github.com/googleapis/google-cloud-cpp/issues/88
 
 ### New Libraries
 
-We are introducing a new client library for [Cloud Run]. While we do not
-anticipate any API changes to this library before declaring it GA, we are
-releasing it early in case it elicits some feedback that requires changes.
+We are introducing new client libraries for [Cloud Run]. While we do not
```

Comment:
"... and [Cloud Batch]." ?

Reply:
nit: the [branding list](https://cloud.google.com/guides/style/branding-list) does not mention Batch / Cloud Batch. So I tend to use whatever the docs say here: 
![image](https://user-images.githubusercontent.com/23088558/178902620-730961c6-efde-42c4-bd3c-7db7894cbfcd.png)

Reply:
Fixed (using @dbolduc text)